src/java.datatransfer/share/classes/java/awt/datatransfer/DataFlavor.java

```diff
@@ -1139,7 +1138,7 @@ public boolean isMimeTypeSerializedObject() {
      * @return the default representation class
      */
     public final Class<?> getDefaultRepresentationClass() {
-        return ioInputStreamClass;
+        return java.io.InputStream.class;
```

Comment:
```suggestion
        return InputStream.class;
```
The class is already imported on line 32.